tests/c/select.newcg.c

```diff
@@ -0,0 +1,56 @@
+// ignore-if: test $YK_JIT_COMPILER != "yk" -o "$YKB_TRACER" = "swt"
+// Run-time:
+//   env-var: YKD_LOG_IR=-:jit-pre-opt,jit-asm
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   env-var: YKD_LOG_JITSTATE=-
+//   stderr:
+//     jitstate: start-tracing
+//     foo
+//     jitstate: stop-tracing
+//     --- Begin aot ---
+//     ...
+//     func main(%arg0: i32, %arg1: ptr) -> i32 {
+//     ...
+//     --- End aot ---
+//     --- Begin jit-pre-opt ---
+//     ...
+//     --- End jit-pre-opt ---
+//     foo
```

Comment:
Would I be an idiot if I said this test doesn't seem to have `print("foo")` anywhere in it?

Reply:
It is I who is the idiot. I pushed a completely not working test (I ran the wrong test to check if things were working :facepalm: ).

Reply:
See [cb11f06](https://github.com/ykjit/yk/pull/1236/commits/cb11f06b971007fd62e947d27bea055f3eee0d2f)

Reply:
That looks a bit better :)